components/automate-chef-io/content/docs/reports.md

```diff
@@ -89,73 +88,82 @@ The download button located to the right of the search bar allows the user to do
 
  Top Environment Failures
-: Represents the amount of compliance scan failures ranked by environment. Hovering over an individual bubble shows the environment and the number of nodes impacted.
+: Represents the amount of compliance scan failures ranked by environment. Hovering over an individual bubble shows the environment name and the number of impacted nodes.
 
 ### Profile Status
 
-The profile status overview provides insight into your system's compliance status from the compliance perspective of Compliance profiles run during scans.
+The _Profile Status_ view provides insight into your system's compliance status from the compliance perspective of Compliance profiles run during scans.
 
 Control Status
-: The control status displays the number of controls in your system, organized by passing, failing, and skipped controls.
- 
+: Displays the number of controls in your system, organized by passing, failing, and skipped controls.
+
 Severity of Control Failures
 : Displays a ranked view of the number and significance of failed controls, sorted into critical, major, and minor control failures.
 
-Test Results Over Time
-: A line graph representing the quantity of controls and Compliance scan results over time. The time-frame can be altered using calendar button in the upper right corner of the chart.
+Control Status Over Time
+: A line graph representing the quantity of controls and Compliance scan results over time. Modify the time-frame using the calendar button in the upper right corner of the chart.
 
 Top Profile Failures
-: Indicates the profiles with the highest failure rate. Hovering over an individual bubble shows the full name of the profile and the number of nodes impacted.
+: Indicates the profiles with the highest failure rate. Hovering over an individual bubble shows the full name of the profile and the number of impacted nodes.
 
 Top Control Failures
-: The Top Control Failures chart illustrates the most frequently failing controls. Hovering over an individual bubble shows the name of the control and the number of nodes impacted.
+: The Top Control Failures chart illustrates the most frequently failing controls. Hovering over an individual bubble shows the name of the control and the number of impacted nodes.
 
-## Switching views
+## Switching Views
 
-In addition to filtering your reports data, you can also switch your views to see compliance report results from the perspective of _Nodes_, _Profiles_, and _Controls_.
+You may also switch your views to see compliance report results from the perspective of _Nodes_, _Profiles_, and _Controls_.
 
 ### Nodes
 
-This view provides more detailed insight into the compliance status of the nodes in your system. This table can be sorted by node name, platform, environment, last scan, and the number of control failures from the most recent compliance scan.
+The _Nodes_ view provides more detailed insight into the compliance status of the nodes in your system.
```

Comment:
This section was bad when I (presumably) wrote it and it is still bad now. Revise to...anything is better.

Reply:
![tenor-179266690](https://user-images.githubusercontent.com/5807053/89831187-643cbc00-db2b-11ea-862a-df85b863543f.gif)